functions/minimissions/fn_bodyBagHandling.sqf

```diff
@@ -0,0 +1,19 @@
+if (isServer) then {
+
+        // body bag scanner
+        ["ace_placedInBodyBag", {
+                params ["_target", "_bodyBag"];
+                diag_log format ["placedInBodyBag _target %1 - bodybag %2", _target, _bodyBag];
+                
+                private _name = name _target;
```

Comment:
Is this reliable? I think ACE uses `ace_common_fnc_getName` for everything involving dead units.

Reply:
This applies tonal uses of `name`